### PR TITLE
Fixes #26244 - Support pulp3 file repos in content views

### PR DESCRIPTION
**Pull request by jlsherrill**

### CODE REVIEW

**Comment by jlsherrill** [comment]
Video going over this PR: https://www.youtube.com/watch?v=cHMCBfkgH-M

File: app/lib/actions/katello/repository/clone_contents.rb [hunk 1]
```diff
@@ -6,22 +6,27 @@ def plan(source_repositories, new_repository, options)
...
           sequence do
             if copy_contents
               source_repositories.each do |repository|
-                plan_action(Pulp::Repository::CopyAllUnits, repository, new_repository,
+                plan_action(PulpSelector,
+                            [Pulp3::Orchestration::Repository::CopyAllUnits, Pulp::Repository::CopyAllUnits],
+                            repository,
+                            SmartProxy.pulp_master,
+                            new_repository,
                             filters: filters, rpm_filenames: rpm_filenames, solve_dependencies: solve_dependencies)
```
Comment (sjha4) [inline app/lib/actions/katello/repository/clone_contents.rb]: This one here seems to not be updated to the new PulpSelctor..

Comment (jlsherrill) [reply app/lib/actions/katello/repository/clone_contents.rb]: updated!

**Comment by jlsherrill** [comment]
rebased ontop of master!

**Comment by parthaa** [comment]
Code wise ack. Just make sure you create an issue to handle composites and address indexing there

**Comment by jlsherrill** [comment]
filed https://projects.theforeman.org/issues/26930

**Pull request merged**